lerna.json

```diff
@@ -3,13 +3,30 @@
     "packages/cli",
     "packages/core",
     "packages/boa",
+    "packages/pipboard-daemon",
+    "packages/pipboard-client",
     "packages/plugins/**/*"
   ],
   "version": "independent",
   "npmClient": "npm",
   "command": {
     "publishConfig": {
       "access": "public"
+    },
+    "bootstrap": {
+      "nohoist": [
+        "node-addon-api",
+        "midway",
+        "egg-scripts",
+        "egg-ci",
+        "midway-bin",
+        "midway-mock",
+        "ts-node",
```

Comment:
ts-node should be shared.

Reply:
Is ts-node required for pipcook? All of codes are compiled to js and run for nodejs:(

Reply:
ts-node is only required by midway.js (daemon)

Reply:
OO:)